diff -urbB kernel/debian.master{,.unpatched}/config/config.common.ubuntu
--- kernel/debian.master/config/config.common.ubuntu	2020-04-20 16:33:49.000000000 +0000
+++ kernel/debian.master.unpatched/config/config.common.ubuntu	2020-04-26 16:18:01.915796595 +0000
@@ -1168,7 +1168,7 @@
 CONFIG_BOOT_PRINTK_DELAY=y
 CONFIG_BPF=y
 CONFIG_BPFILTER=y
-CONFIG_BPFILTER_UMH=m
+CONFIG_BPFILTER_UMH=y
 CONFIG_BPF_EVENTS=y
 CONFIG_BPF_JIT=y
 CONFIG_BPF_KPROBE_OVERRIDE=y
