diff -urbB kernel.unpatched/debian.master/config/config.common.ubuntu kernel/debian.master/config/config.common.ubuntu
--- kernel.unpatched/debian.master/config/config.common.ubuntu  2020-04-01 00:47:16.499541862 +0000
+++ kernel/debian.master/config/config.common.ubuntu    2020-04-01 00:47:44.554871398 +0000
@@ -1170,7 +1170,7 @@
 CONFIG_BOOT_PRINTK_DELAY=y
 CONFIG_BPF=y
 CONFIG_BPFILTER=y
-CONFIG_BPFILTER_UMH=m
+CONFIG_BPFILTER_UMH=y
 CONFIG_BPF_EVENTS=y
 CONFIG_BPF_JIT=y
 CONFIG_BPF_KPROBE_OVERRIDE=y
